flink-runtime/src/test/java/org/apache/flink/runtime/highavailability/FileSystemJobResultStoreContractTest.java

```diff
@@ -35,6 +35,6 @@ public class FileSystemJobResultStoreContractTest implements JobResultStoreContr
     @Override
     public JobResultStore createJobResultStore() throws IOException {
         Path path = new Path(temporaryFolder.toURI());
-        return new FileSystemJobResultStore(path.getFileSystem(), path, false);
+        return new FileSystemJobResultStore(path.getFileSystem(), path, false, Runnable::run);
```

Comment:
nit: `Executors.directExecutor()` would be the usual Executor for direct execution.

Reply:
Fixed.